hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/signature/TestStringToSignProducer.java

```diff
@@ -269,4 +271,60 @@ public void testValidateCanonicalHeaders(
 
     Assert.assertEquals(expectedResult, actualResult);
   }
+
+  @Test
+  public void testVirtualStyleAddressURI() throws Exception {
```

Comment:
Can we rename the other test in this class to `testPathStyleAddressURI` to clarify the difference?